modules/server_license.py

```diff
@@ -0,0 +1,20 @@
+from prometheus_client import Metric
+
+
+class UcsServerLicense:
+    def __init__(self, handles):
+        self.handles = handles
+
+    def collect(self):
+        print("Get license info of ports ")
+        license_state_metric = Metric('ucs_license', 'Information about port license', "gauge")
+        for server, handle in self.handles.items():
```

Comment:
and here it comes in handy, if you have the handles stored in a `BaseCollector`

Reply:
Added handles  in UcsmCollector.